src/core/nitro.ts

```diff
@@ -35,6 +36,12 @@ export async function createNitro(
     },
   };
 
+  // Set Nitro context for `useNitro`
```

Comment:
```suggestion
  // Set Nitro context for `useNitro`. We should verify if it exists because prerender will call the `createNitro` which will reset the context and throw an error.
```